symfony/config/packages/twig.php

```diff
@@ -8,11 +8,10 @@
 $container->loadFromExtension('twig', [
     'globals' => [
         'isStable' => $env->isStable(),
-        'isDvType' => '',
-        'reportKitUrl' => '',
+        'reportKitUrl' => $env->configuration['reportKitUrl'],
         'assetVer' => $env->values['assetVer'],
         'awardeeOrganization' => '',
-        'google_analytics_property' => '',
+        'google_analytics_property' => $env->configuration['google_analytics_property'],
```

Comment:
As mentioned above in the config file, I think this is where we need to make an adjustment to use an empty value if a non-critical configuration item like these isn't set, instead of breaking due to an unset array key (I'm assuming that's the reason this breaks).